rllib/BUILD

```diff
@@ -846,6 +846,34 @@ py_test(
     srcs = ["models/tests/test_distributions.py"]
 )
 
+# --------------------------------------------------------------------
+# Optimizers
+# rllib/optimizers/
+#
+# Tag: optimizers
+# --------------------------------------------------------------------
+
+py_test(
+    name = "test_optimizers",
+    tags = ["optimizers"],
+    size = "large",
```

Comment:
Is this really large (>5 min)?

Reply:
Changed to "medium".

Reply:
Nope, must to be "large". Times out in travis sometimes, otherwise.